ynr/apps/elections/uk/templates/candidates/person-view.html

```diff
@@ -219,7 +219,14 @@ <h2>Improve this data!</h2>
       <p>Please do add extra details about this candidate – it only takes a moment.</p>
       {% if user.is_authenticated %}
         <a href="{% url 'person-update' person_id=person.id %}" class="button">Edit candidate</a>
-        <a href="{% url 'photo-upload' person.id %}" class="button">Upload candidate photo</a>
+        {% if person.queued_image %}
+          <p>{{ person.name }} has a photo that needs to be reviewed</p>
```

Comment:
Yes I agree it could be confusing/frustrating for users to just see this message without having the power to see it/change it. So perhaps a view that allows them to see the image and change it could be an alternative - think this was mentioned before but not sure if we decided to leave for now?

Reply:
Perhaps a MVP alternative is to still have the link to the upload page, as that displays a bit more information about there being an uploaded image pending?

Reply:
Only users with permission to review will ever see the mod queue (here or in the notifications banner at top). After testing this out with a dummy user with just uploading permission, I think it makes the most sense to simply add the message about existing images within the `{% if user_can_review_photos %}` so that messages better match permissions.